terraform-install/querypie/manage-ec2-ssm.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Terraform 출력에서 EC2 인스턴스 ID 추출
+INSTANCE_ID=$(terraform show | grep -Eo '\bi-[0-9a-fA-F]{8,17}\b' | sort | uniq)
+
+# 인스턴스 ID가 없는 경우 스크립트 종료
+if [ -z "$INSTANCE_ID" ]; then
+    echo "No EC2 instance ID found in Terraform output."
+    exit 1
+fi
+
+# 추출된 인스턴스 ID 출력
+echo "Found EC2 instance ID: $INSTANCE_ID"
+
+# 사용자에게 인스턴스를 중지할지 시작할지 묻기
+if [ $# -eq 0 ]; then
+    echo "No action specified. Usage: $0 {start|stop|connect}"
+    exit 1
+fi
+
+ACTION=$1
```

Comment:
```suggestion
ACTION=${1:-}
```

지금은 bash shell option, `-o nounset` 이 적용되지 않았는데요.
`set -o nounset` 옵션을 사용한다면, 기본값을 주는 것이 좋을 것 같습니다.

Reply:
해당 shell 옵션을 추가하도록 하겠습니다. 감사합니다.